README.md

```diff
@@ -1,2 +1,79 @@
-# authentication-members-only-ror
-Using the Devise gem to have a first approach at authentication in rails.
+# members-only - Rails
+
+> Using the Devise gem to have a first approach at authentication in rails.
+
+## Models created
+
+- User
+- Post
+
+## Built With
+
+- Ruby 2.5.1
+- Rails 6.0.2
+
+
+## Screenshot
+
+![image](https://user-images.githubusercontent.com/5160907/81008694-f771a180-8e18-11ea-901a-c2ec4be49735.png)
+
+
+## Usage
+
+This is the main page, you will be able to create posts until you are logged in.
```

Comment:
This should read, _This is the main page, you will **not** be able to create posts until you are logged in._

Reply:
Fixed, thank you